lib/uksched/Makefile.uk

```diff
@@ -14,4 +14,4 @@ LIBUKSCHED_SRCS-y += $(LIBUKSCHED_BASE)/thread.c
 LIBUKSCHED_THREAD_FLAGS-$(call gcc_version_ge,8,0) += -Wno-cast-function-type
 LIBUKSCHED_SRCS-y += $(LIBUKSCHED_BASE)/extra.ld
 
-UK_PROVIDED_SYSCALLS-$(CONFIG_LIBUKSCHED) += sched_yield-0
+UK_PROVIDED_SYSCALLS-$(CONFIG_LIBUKSCHED) += sched_yield-0 sched_getaffinity-3 sched_setaffinity-3
```

Comment:
Add these on separate lines (like you did in the `lib/posix-mmap/Makefile.uk` file)